support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/ValidatePasswordResetTokenActionTests.java

```diff
@@ -24,22 +33,50 @@
 @EnabledIfListeningOnPort(port = 25000)
 @Tag("Mail")
 class ValidatePasswordResetTokenActionTests extends BasePasswordManagementActionTests {
+
+    private static final String TOKEN = "token";
+
+    @BeforeEach
+    public void onSetUp() {
+        val request = new MockHttpServletRequest();
+        request.setRemoteAddr("185.86.151.11");
+        request.setLocalAddr("185.88.151.11");
+        val clientInfo = ClientInfo.from(request);
+        ClientInfoHolder.setClientInfo(clientInfo);
+    }
+
     @Test
     void verifyInvalidTicket() throws Throwable {
-        val context = MockRequestContext.create();
+        val context = MockRequestContext.create(applicationContext);
         context.setParameter(PasswordManagementService.PARAMETER_PASSWORD_RESET_TOKEN, UUID.randomUUID().toString());
         assertEquals(CasWebflowConstants.TRANSITION_ID_INVALID_PASSWORD_RESET_TOKEN, validatePasswordResetTokenAction.execute(context).getId());
     }
 
+    @Test
+    void verifyGoodTicket() throws Throwable {
+        val query = PasswordManagementQuery.builder()
+                .username("casuser").email("casuser@email.com").build();
+        val token = passwordManagementService.createToken(query);
+
+        val id = UUID.randomUUID().toString();
+        val ticket = new TransientSessionTicketImpl(id, NeverExpiresExpirationPolicy.INSTANCE,
+                CoreAuthenticationTestUtils.getService(), Map.of(TOKEN, token));
+        ticketRegistry.addTicket(ticket);
+
+        val context = MockRequestContext.create(applicationContext);
+        context.setParameter(PasswordManagementService.PARAMETER_PASSWORD_RESET_TOKEN, id);
```

Comment:
```suggestion
        context.setParameter(PasswordManagementService.PARAMETER_PASSWORD_RESET_TOKEN, ticket.getId());
```

Reply:
OK.

Reply:
I have updated both PRs to take into account your latest feedbacks.